CMakeLists.txt

```diff
@@ -72,6 +72,10 @@ endif()
 add_executable(ninja src/ninja.cc)
 target_link_libraries(ninja PRIVATE libninja libninja-re2c)
 
+if(MINGW)
+target_compile_definitions(ninja PRIVATE __USE_MINGW_ANSI_STDIO=1)
```

Comment:
Can you move this into `target_compile_definitions(libninja PRIVATE _WIN32_WINNT=0x0601)`?